test/server/routes/insertTaskRoute.js

```diff
@@ -4,7 +4,7 @@ const route = require('../../../server/app');
 const connect = require('../../../server/database/config/connection');
 
 const selectUserScrumProj = () => {
-  const getUser = 'SELECT projects.*, scrums.project_id as scrum_id, users_projects.user_id as user_id FROM users_projects join projects ON projects.id = users_projects.project_id inner join scrums on scrums.project_id = users_projects.project_id';
```

Comment:
assigned_to is not the user_id so I changed this query to make the test pass @Angham116